blockchain/accesstokens.go

```diff
@@ -2,111 +2,54 @@ package blockchain
 
 import (
 	"context"
+	"encoding/hex"
 	"encoding/json"
+	"fmt"
 
-	"github.com/bytom/blockchain/accesstoken"
 	"github.com/bytom/errors"
-	//	"github.com/bytom/net/http/authz"
-	"github.com/bytom/net/http/httpjson"
 )
 
-/*
-const (
-	defGenericPageSize = 100
-)
-*/
 var errCurrentToken = errors.New("token cannot delete itself")
 
-func (bcr *BlockchainReactor) createAccessToken(ctx context.Context, x struct{ ID, Type string }) (*accesstoken.Token, error) {
-	token, err := bcr.accesstoken.Create(ctx, x.ID, x.Type)
+func (a *BlockchainReactor) createAccessToken(ctx context.Context, x struct{ ID, Type string }) interface{} {
```

Comment:
Why don't use `bcr` for `BlockchainReactor`?

Reply:
bcr is better